# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cloud-utils/hotfix-toml.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cloud-utils-0.34.orig/pyproject.toml	2026-06-26 14:08:45.000000000 +0200
+++ cloud-utils-0.34/pyproject.toml	2026-07-26 06:34:54.415071324 +0200
@@ -1,4 +1,6 @@
 [project]
+name = "cloud-utils"
+version = "0.34"
 requires-python = ">= 3.13"
 
 [tool.codespell]
